lib/ts/recipe/passwordless/components/features/linkClickedScreen/index.tsx

```diff
@@ -52,15 +53,54 @@ const LinkClickedScreen: React.FC<PropType> = (props) => {
                     });
                 }
 
-                const response = await props.recipe.recipeImpl.consumeCode({
-                    preAuthSessionId,
-                    linkCode,
-                    userContext,
-                });
+                let response:
+                    | {
+                          status: "OK";
+                          createdUser: boolean;
+                          user: PasswordlessUser;
+                          fetchResponse: Response;
+                      }
+                    | {
+                          status: "INCORRECT_USER_INPUT_CODE_ERROR" | "EXPIRED_USER_INPUT_CODE_ERROR";
+                          failedCodeInputAttemptCount: number;
+                          maximumCodeInputAttempts: number;
+                          fetchResponse: Response;
+                      }
+                    | {
+                          status: "RESTART_FLOW_ERROR";
+                          fetchResponse: Response;
+                      }
+                    | undefined;
```

Comment:
Do you need to define the full types here?

Reply:
I was having this weird issue where typescript would complain about `createdUser` not being present even if status was OK. Adding the types resolved this issue.